lib/widgets/inbox.dart

```diff
@@ -327,6 +329,11 @@ class _AllDmsHeaderItem extends _HeaderItem {
     pageState.allDmsCollapsed = !collapsed;
   }
   @override Future<void> onRowTap() => onCollapseButtonTap(); // TODO open all-DMs narrow?
+
+  @override Future<void> onLongPress() async {
+     // Direct message sections don't have specific long-press actions,
+      // unlike stream sections which show channel action sheets
+  }
```

Comment:
Instead of a handler function with an empty body, this should just be null; see the `TODO(a11y)` you added (via copy-paste) to `showChannelActionSheet`. And instead of this comment, say `// TODO(#1272) action sheet for DM conversation`.